socless/events.py

```diff
@@ -173,7 +173,7 @@ def __init__(
         self, initial_event: Union[InitialEvent, None] = None, **initial_event_args
     ) -> None:
         """Methods:
-        `as_event_table_item`: returns formatted event data for input to events_table
+        `to_EventTableItem`: returns formatted event data for input to events_table
         `deduplicate_and_update_dedup_table`: check dedup_table & event_table to see if this event exists, mutate self.metadata and update dedup_table accordingly
         `put_in_events_table`: put event in events_table (does NOT deuplicate automatically)
         `start_playbook` :
```

Comment:
Documenting these here seems a little misleading as it gives the impression that they're arguments to the initializer but they're really `methods` on functions. 
Places we can move the docs to:
- Docstrings underneath `CreateEvents`
- Docstring for each function